openshift/e2e-tests-openshift.sh

```diff
@@ -184,6 +184,29 @@ function run_e2e_tests(){
     --imagetemplate "$TEST_IMAGE_TEMPLATE" \
     --resolvabledomain "$(ingress_class)" || failed=1
 
+  # Enable leader election
+  oc -n "$SERVING_NAMESPACE" patch configmap/config-leader-election --type=merge \
```

Comment:
Shouldn't we do this via the HighAvailability setting in the KnativeServing we apply? More so, this should even be already done as we default to 2, we might need to update the knative-operator though.

Reply:
I was not sure if this setting is applied in this concrete repository. I was planning to do this when running the test suite as part of serverless-operator repo. But is this setting applied here? Perhaps we might need to update the knative-operator as you said. Please advise.

Reply:
Yeah the same stuff should be happening here as we setup using a particular version of serverless-operator. We'll need to use Serverless Operator v1.6.0 + Knative Serving 0.13.x to get the setting I think.

Reply:
I'm a bit lost on how to update the deps, can you advise please? Do I need to add new csv to https://github.com/openshift/knative-serving/blob/release-v0.13.2/openshift/olm/knative-serving.catalogsource.yaml ?

Reply:
Looks like you only need to update https://github.com/openshift/knative-serving/blob/07e14b015d44609fad4d7a2c251953d34bf871a3/openshift/olm/knative-serving.catalogsource.yaml#L2774 to point to a newer knative-serving-operator version, I think. We should clean that file up to only contain one version at some point.

Reply:
Thanks!